docusaurus.config.js

```diff
@@ -89,6 +89,12 @@ const config = {
             position: 'left',
             label: 'Guides',
           },
+          // {
+          //   type: "docSidebar",
+          //   position: "left",
+          //   sidebarId: "learn",
+          //   label: "Learn",
+          // },
```

Comment:
please remove, dont comment out

Reply:
Removed, needed this for the new section of the docs that is currently WIP